resources/assets/less/bem/forum-topic-logs-table.less

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+.forum-topic-logs-table {
+    width: 100%;
+
+    &__body {
+        font-size: @font-size--title-small;
+    }
+
+    &__date {
```

Comment:
the action column can probably have 100% width with other columns having some padding and `white-space: nowrap`